server/analyzer/init.go

```diff
@@ -15,23 +15,80 @@
 package analyzer
 
 import (
+	"fmt"
+
 	"github.com/dolthub/go-mysql-server/sql/analyzer"
 )
 
 // Init adds additional rules to the analyzer to handle Doltgres-specific functionality.
 func Init() {
 	// IDs are basically arbitrary, we just need to ensure that they do not conflict with existing IDs
+	analyzer.AlwaysBeforeDefault = append(analyzer.AlwaysBeforeDefault,
+		analyzer.Rule{Id: 1000, Apply: TypeSanitizer},
```

Comment:
Should define consts for these ids

Reply:
Also generate strings for the consts you define, as GMS does. Makes reading analyzer debug output much easier.

Reply:
Can't do that without modifying GMS sadly